code/controllers/subsystem/ticker.dm

```diff
@@ -156,10 +156,10 @@ SUBSYSTEM_DEF(ticker)
 				//lobby stats for statpanels
 			if(isnull(timeLeft))
 				timeLeft = max(0,start_at - world.time)
-			totalPlayers = LAZYLEN(GLOB.new_player_list)
+			totalPlayers = LAZYLEN(GLOB.new_player_list)
```

Comment:
This file only has white space changes. Please undo.

Reply:
That's strange, I didn't touch that file. 

![2020-05-14 05_53_01-ticker dm - tgstation - Visual Studio Code](https://user-images.githubusercontent.com/36066032/81931364-3638f180-95a7-11ea-8ee3-2f77271b12d9.png)

This is what I see.